README.md

```diff
@@ -99,6 +99,14 @@ apk add flawz
 brew install flawz
 ```
 
+### Nix
+
+**flawz** is available for Nix via [nixpkgs/master](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/fl/flawz/package.nix). To use it, simply run:
+
+```sh
+nix run github:NixOS/nixpkgs/master#flawz
```

Comment:
Is this installing the package? Or how does it work?

Ref: https://github.com/orhun/kmon?tab=readme-ov-file#nixpkgs

Reply:
ah, I use the new (experimental) CLI, which is way more simple, but yeah sure, it's still experimental, so I'll add the legacy way 👍🏻

Reply:
> Is this installing the package?

no, it's for running it directly, which's the recommend way (in my mind), at least until it lands on the `nixos-unstable` branch

Reply:
I see, can we mention this detail too?